taskcluster/scripts/toolchain/android-gradle-dependencies/nexus.xml

```diff
@@ -0,0 +1,413 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is a copy of https://searchfox.org/mozilla-central/source/taskcluster/scripts/misc/android-gradle-dependencies/nexus.xml. I didn't make any changes.